frontend/src/libs/components/app/app.tsx

```diff
@@ -37,6 +38,7 @@ const App: React.FC = () => {
         <div>
           <RouterOutlet />
         </div>
+        <MeditationPlayer />
```

Comment:
Please don't mess up with app component. I understand that you want to see your component, but you can remove it before creating pr)